src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ManageApprenticesController.cs

```diff
@@ -23,29 +21,40 @@ public ManageApprenticesController(ICommitmentsService commitmentsService, ICrea
             _createCsvService = createCsvService;
         }
 
-        public async Task<IActionResult> Index(uint providerId)
+        [Route("manage", Name = RouteNames.ManageApprentices)]
```

Comment:
I mentioned this to Scott, but there's no need for "v2" in the route; ProviderCommitments _is_ v2.

Reply:
we had v2 so that we could run it side by side while developing the page, however if the base url is completely different then there is no need for the v2

Reply:
ok i'll remove it

Reply:
done